### ISSUE TITLE
Subcutaneous fatty lesions with a "blush like" signal on MRI

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10001]
**Preferred term label:**
Subcutaneous fatty lesions with a "blush like" signal on MRI

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adiposis dolorosa

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000056
- label: Superficial blush-like subcutaneous fatty lesions
- parents: [Abnormality of subcutaneous fat tissue (HP:0001001)](https://hpo.jax.org/app/browse/term/HP:0001001)
- pmid: PMID:23809264
- def: Multiple, oblong, fatty lesions in the superficial subcutaneous fatty tissue. They are mostly under 2 cm in long axis diameter. They demonstrate nodular (blush-like) increased fluid signal at unenhanced MRI and are markedly hyperechoic at ultrasound.
- comment: These lesions are characteristicv of Adiposis dolorosa (Dercum's disease).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10001
- synonyms:

**Comment by MickeySegal** [comment]
You already used this HPO number:

https://github.com/obophenotype/human-phenotype-ontology/issues/10084#issuecomment-1871270193

ID: HP:6000056
label: Breast microcalcifications
